packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx

```diff
@@ -30,9 +30,13 @@ const CapturePhoto: React.FC<CapturePhotoProps> = ({ initialState, onCapturePhot
 
   return (
     <div style={{ display: 'flex', alignItems: 'center' }}>
-      <div style={{ maxWidth: '64px' }}>
+      <button
+        type="button"
+        onClick={showCam}
+        style={{ border: 'none', maxWidth: '64px', padding: 0, margin: 0, background: 'none', cursor: 'pointer' }}
```

Comment:
This should be in a style (scss) file

Reply:
@brandones,
I have added a new file named `capture-photo.scss` and added a class named `.buttonCssReset`

There was no existing css file for this component nor it was importing styles from any other css file. So I created a new one